homework/02LessonTwo.js

```diff
@@ -126,7 +188,7 @@ function whichSpecies(character) {
 //
 // In your own words, explain what conditionals do.
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: Conditionals (e.g., if, else, if else) are used to make different actions happen based on different conditions. 
```

Comment:
Hi Kimberly! To be more specific, conditionals are used to make different actions happen based on the Boolean value that is obtained from evaluating the condition: example: if (number % 2 === 0) will yield true or false, and an action will happen based on that Boolean value.